tutorials/performance/using_multiple_threads.rst

```diff
@@ -47,6 +47,89 @@ To create a thread, use the following code:
     func _exit_tree():
         thread.wait_to_finish()
 
+ .. code-tab:: cpp C++ .H File
+    #ifndef MULTITHREADINGDEMO_H
+    #define MULTITHREADINGDEMO_H
+    
+    #include <godot_cpp/classes/node.hpp>
+    #include <godot_cpp/classes/thread.hpp>
+    
+    namespace godot {
+    
+    	class MultithreadingDemo : public Node {
+    		GDCLASS(MultithreadingDemo, Node)
+    
+    	private:
+    		Ref<Thread> worker;
+    
+    	protected:
+    		static void _bind_methods();
+    		void _notification(int p_what);
+    
+    	public:
+    		MultithreadingDemo();
+    		~MultithreadingDemo();
+    
+    		void demo_threaded_function();
+    	};
+    }
+    #endif
+
+ .. code-tab:: cpp C++ .CPP File
+    #include "MultithreadingDemo.h"
```

Comment:
```suggestion

    #include "multithreading_demo.h"

```
The files should match the style for c++ files 🙂

And a space between the header and the rest of the includes